File: Backend/Easylearner/src/main/kotlin/hu/onlab/easylearner/model/entities/Lesson.java
package hu.onlab.easylearner.model.entities;


import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "LESSON")
public class Lesson {
    @Id
    Integer Id;

    @Basic
    @Column(name = "TEACHERID")
    Integer TeacherId;

    @Basic
    @Column(name = "STUDENTID")
    Integer StudentId;

    @Basic
    @Column(name = "INFO")
    String Info;

    @Basic
    @Column(name = "STARTTIME")
    Long StartTime;

    @Basic
    @Column(name = "ENDTIME")
    Long EndTime;

    @Basic
    @Column(name = "LONGITUDE")
    String Longitude;

    @Basic
    @Column(name = "LATITUDE")
    String Latitude;

    @Basic
    @Column(name = "PAYMENT")
    Integer Payment;

    @Basic
    @Column(name = "LEVELID")
    Integer LevelId;

    @Basic
    @Column(name = "TOPICID")
    Integer TopicId;

    @Basic
    @Column(name = "OWNERID")
    Integer OwnerId;
}

File: Android/app/src/main/java/hu/bme/aut/android/easylearner/model/Lesson.kt
package hu.bme.aut.android.easylearner.model

import android.os.Parcelable
import java.io.Serializable

class Lesson : Serializable{
    val id : Int = 0
    val teacherId : Int = 0
    val studentId : Int = 0
    val teacherName : String = ""
    val studentName : String = ""
    val info : String = ""
    val startTime : Long = 0
    val endTime : Int = 0
    val longitude : String = ""
    val latitude : String = ""
    val payment : Int = 0
    val levelName : String = ""
    val topicName : String = ""
}